.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to the Salesforce CLI
+Welcome, and thank you for your interest in contributing to the Salesforce CLI!
+
+There are multiple ways you can contribute:
+
+### Report an issue
+Did you find a bug? First, make sure the bug wasn't already reported by searching on the `issues` section: https://github.com/forcedotcom/cli/issues.
+If you don't find an issue, then open a new one and follow the bug template. Minimal repros are highly encouraged as these help us to quickly identify the root cause of the issue.
```

Comment:
`If you don't find an issue,` --> `If you cannot find a related issue.`
`Minimal repros are` --> `Providing a minimal reproduction is `
Sidenote: We should create a `.md` doc on how to create a good reproduction.